src/sagas/helpers.js

```diff
@@ -275,3 +279,54 @@ export function getNetworkSettings(state) {
   // has precedence, once it indicates a custom network.
   return STORE.getItem(networkSettingsKeyMap.networkSettings) ?? state.networkSettings;
 }
+
+/**
+ * A request abstraction that applies a progressive retry strategy.
+ * One can define how many retries it should make or use the default value.
+ *
+ * @param {Promise<any>} request The async callback function to be executed.
+ * @param {number} maxRetries The max retries allowed, with default value.
+ * Notice this param should be at least 2 to make sense.
```

Comment:
Why? The minimum should be 1.

If the parameter name contains 'retries', then 1 should be ok. You try once, then if it fails, you retry once.
 
However the implementation is different, if you set `maxRetries = 5`, then you try once and retries 4 times only, which should be 5.

Reply:
Oh. Got it. You right, the implementation should account only the retries.

Reply:
Done at https://github.com/HathorNetwork/hathor-wallet-mobile/pull/444/commits/c552aa618f7da09b9353666901f6bfe4d6ca441e